core/trino-main/src/main/java/io/trino/server/DynamicFilterService.java

```diff
@@ -229,7 +229,7 @@ public boolean isCollectingTaskNeeded(QueryId queryId, PlanFragment plan)
     {
         DynamicFilterContext context = dynamicFilterContexts.get(queryId);
         if (context == null) {
-            // query has been removed or not registered (e.g dynamic filtering is disabled)
+            // query has been removed or not registered (e.g. dynamic filtering is disabled)
```

Comment:
There're other usage of `e.g `

Reply:
Updated all java files where "e.g" appears. 

`git grep -n 'e\.g ' | grep --color=auto '\.java:' | grep --color=auto 'e\.g '`

```
core/trino-main/src/main/java/io/trino/execution/scheduler/UniformNodeSelector.java:314:            // among nodes in a cluster won't be fully uniform (e.g because hash function with non-uniform
core/trino-main/src/main/java/io/trino/failuredetector/HeartbeatFailureDetector.java:182:                    // TODO: distinguish between process unresponsiveness (e.g GC pause) and host reboot
core/trino-main/src/main/java/io/trino/operator/BasicWorkProcessorOperatorAdapter.java:27: * that do not require special input handling (e.g streaming operators).
core/trino-main/src/main/java/io/trino/operator/WorkProcessorOperatorAdapter.java:25: * that require customization of input handling (e.g aggregation operators that want to skip extra
core/trino-main/src/main/java/io/trino/operator/join/LookupJoinPageBuilder.java:153:        // Lazy blocks (e.g used in filter condition) could be loaded during filter evaluation.
core/trino-main/src/main/java/io/trino/operator/join/unspilled/LookupJoinPageBuilder.java:155:        // Lazy blocks (e.g used in filter condition) could be loaded during filter evaluation.
core/trino-main/src/main/java/io/trino/operator/project/MergePages.java:232:            // TODO: provide better heuristics there, e.g check if last produced page was materialized
core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/PreAggregateCaseAggregations.java:334:            // only cumulative aggregations (e.g that can be split into aggregation of aggregations) are supported
core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/PreAggregateCaseAggregations.java:434:        // cumulative aggregation function (e.g aggregation of aggregations)
core/trino-spi/src/main/java/io/trino/spi/block/LazyBlockLoader.java:20:     * (in case of when loaded blocks are nested, e.g for structural types).
plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/writer/S3TransactionLogSynchronizer.java:136:            // extra check if target file did not appear concurrently; e.g due to conflict with TL writer which uses different synchronization mechanism (like DB)
plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveNodePartitioningProvider.java:83:        // to shuffled nodes (e.g "1 -> node2, 2 -> node1, 3 -> node2, 4 -> node1, ...").
plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveNodePartitioningProvider.java:85:        // (e.g "(part1, bucket1) -> 1234, (part1, bucket2) -> 1235, ...").
```